opr-core/src/model/getupdatetimestamp.ts

```diff
@@ -0,0 +1,5 @@
+import {Offer} from 'opr-models';
```

Comment:
Once upon a time, there was a getUpdateTimestamp method that became so popular that everyone started using it. But then the lawyers realized that the file did not have a license header, and they were very sad.

(This short story brought to you by GPT-3)

Reply:
Also adorable. Done.